portfolio/src/main/webapp/index.html

```diff
@@ -10,9 +11,19 @@
     <script src="script.js"></script>
   </head>
   <body>
+    <nav class="navbar navbar-light bg-primary navbar-extend-lg">
+        <ul class="nav navbar-nav navbar-left">
```

Comment:
nit: This block can be un-indented two spaces

Reply:
Fixed!